.Gemfile-rest-client-1.8.rb

```diff
@@ -0,0 +1,11 @@
+# Allow travis to test rest-client 1.x.
```

Comment:
I'm testing both, see .travis.yml.
I'm all for bumping to rest-client >= 2.0 and dropping these at some later point.